### PR TITLE
feat: migrate to jina 3

### PR DESCRIPTION
**Pull request by Hippopotamus0308**

### CODE REVIEW

File: gpu_requirements.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+torch==1.9.0
+torchvision
+pytorch-ignite==0.3.0
```
Comment (samsja) [inline gpu_requirements.txt:3]: not needed, I guess it is only for training right ?

Comment (Hippopotamus0308) [reply gpu_requirements.txt]: right, I've removed it

File: .github/workflows/cd.yml [hunk 2]
```diff
@@ -4,16 +4,15 @@ on:
...
-      - name: Checkout
-        uses: actions/checkout@v2
-      - id: push
-        run: jina hub push --force ${{ secrets.HUBBLE_UUID }} --secret ${{ secrets.HUBBLE_SECRET }} .
+  call-external:
+    uses: jina-ai/workflows-executors/.github/workflows/cd.yml@v0.3
+    with:
+      event_name: ${{ github.event_name }}
+    secrets:
+      secret: ${{ secrets.EXECUTOR_SECRET }}
```
Comment (nan-wang) [inline .github/workflows/cd.yml:18]:
```suggestion
      secret: ${{ secrets.HUBBLE_SECRET }}
```

**Review by nan-wang — CHANGES_REQUESTED** [review]
remove `executor/audio_clip/.DS_Store`

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**